src/pages/index.jsx

```diff
@@ -19,6 +21,7 @@ export default function HomePage() {
                     العربية
                 </Link>
             </div>
+            <HeroSection />
```

Comment:
Just a side note : we agreed to not touch the `index.jsx` and just add the wrapper component in the end .. however it's fine since i didn't modify this file (I hope so.. i don't remember :rofl:  )